src/som/vm/Shell.java

```diff
@@ -69,8 +71,8 @@ public SAbstractObject start() {
 
     Universe.println("SOM Shell. Type \"quit\" to exit.\n");
 
-    // Create a fake bootstrap frame
-    currentFrame = interpreter.pushNewFrame(bootstrapMethod);
+    // fetch the bootstrap frame
```

Comment:
Is this comment helpful? Perhaps rename `frame` to `bootstrapFrame` so it's explicit in the code?